app/services/analytics_events.rb

```diff
@@ -5182,14 +5212,15 @@ def user_registration_2fa_setup_visit(
   end
 
   # User registration has been handed off to agency page
-  # @param [Boolean] ial2
-  # @param [Integer] ialmax
-  # @param [String] service_provider_name
-  # @param [String] page_occurence
-  # @param [String] needs_completion_screen_reason
+  # @param [Boolean] ial2 Whether the user registration was for a verified identity
+  # @param [Integer] ialmax Whether the user registration was for an IALMax request
+  # @param [String] service_provider_name The friendly name of the service provider
+  # @param ['account-page','agency-page'] page_occurence Where the user concluded registration
+  # @param ['new_sp','new_attributes','reverified_after_consent'] needs_completion_screen_reason The
```

Comment:
it doesn't matter but I notice we are using double quotes for other string values, single quotes here

Reply:
Ah I think I saw another method where we used single quotes which is why I used them here, so we're clearly a big mess of inconsistency 🙃

Reply:
If you're referring to lines like:

```
# @param ["authentication","reauthentication","confirmation"] context User session context
```

Those were copied from other methods, hence the internal inconsistency.

I may follow-up to do a bulk normalization, though I feel like it's inevitable we'll regress to having some inconsistency, unless we care to enforce it somehow.

Reply:
I took a look at [rubocop-yard](https://github.com/ksss/rubocop-yard) but unfortunately, it doesn't appear to handle string literals at all. Either way, a task for another time